cli/fathom_web/test/test_pick.py

```diff
@@ -61,3 +61,50 @@ def test_end_to_end(tmp_path):
     # Make sure we didn't lose any files
     files_in_directories = {file.name for file in files_in_source} | {file.name for file in files_in_destination}
     assert {'1.html', '2.html', '3.html'} == files_in_directories
+
+
+def test_resource_directory_path_collision(tmp_path):
+    """
+    Ensure an exception is raised when moving a resource directory
+    if that directory already exists in the destination directory.
+    """
+    # Make temporary source and destination directories
+    source = tmp_path / 'source'
+    source.mkdir()
+    destination = tmp_path / 'destination'
+    destination.mkdir()
+
+    # Add the file to the source directory
+    file_1 = source / '1.html'
+    file_1.write_text('I am file 1')
+
+    # Add the resource directory for our file
+    resources = source / 'resources'
+    resources.mkdir()
+    file_1_resources = resources / '1'
+    file_1_resources.mkdir()
+    file_1_resource_1 = file_1_resources / '1.png'
+    file_1_resource_1.write_text('I am resource 1 for file 1')
```

Comment:
Since we aren't asserting anything about the file contents, does it do us any good to write any?

Reply:
It does us no good. I forgot about the method `Path.touch()` to make empty files. I have that change queued up.